### ISSUE TITLE
WebSockets and mTLS

### ISSUE THREAD

**Issue by astonbitecode** [a85/POSTMan-Chrome-Extension#11865]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

I have added my CA certificate as needed and an entry in client certificates defining the appropriate CRT and KEY files for a target host. The HTTPS requests work without issues, using mTLS.

However, the WebSocket connections fail when mTLS is in place. Here is the error I get:

```
Error: write EPROTO 67861384:error:10000412:SSL routines:OPENSSL_internal:SSLV3_ALERT_BAD_CERTIFICATE:../../../../src/third_party/boringssl/src/ssl/tls_record.cc:594:SSL alert number 42
Handshake Details
Request URL: https://localhost:9543/wswork
Request Method: GET
Request Headers
Sec-WebSocket-Version: 13
Sec-WebSocket-Key: UxVi2kzWRrMvi8bAXpGPOQ==
Connection: Upgrade
Upgrade: websocket
Sec-WebSocket-Extensions: permessage-deflate; client_max_window_bits
Host: localhost:9543
```
Please note that the WebSockets are connecting without issue when not using mTLS:

```
Handshake Details
Request URL: https://localhost:8443/wswork
Request Method: GET
Status Code: 101 Switching Protocols
Request Headers
Sec-WebSocket-Version: 13
Sec-WebSocket-Key: OjwwNp0LeQplCEQ16xbdZw==
Connection: Upgrade
Upgrade: websocket
Authorization: Basic YWRtaW46Q2hnTWVOT1c=
Sec-WebSocket-Extensions: permessage-deflate; client_max_window_bits
Host: localhost:8443
Response Headers
Sec-WebSocket-Extensions: permessage-deflate
Connection: Upgrade
```

### Steps To Reproduce

1. Setup the certificates as needed for mTLS
2. Try to connect to a WebSocket using mTLS

### Operating System

Windows

### Postman Version

10.12.8

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by marinator86** [comment]
same for me. I also cannot find docs about this

**Comment by mefly2012** [comment]
yes ,same for me too，macos ，postman version 10.20.10

**Comment by iliyan85** [comment]
The issue still exists.
Postman v10.24.17

**Comment by parthverma1** [comment]
This should now be fixed as part of v11.0.12+. Please update your app. Feel free to re-open the issue if you're still facing the same problem.

**Comment by iliyan85** [comment]
Sorry, but I tested with version v11.1.14 and the problem still exists.
Nginx is installed on the server side. The log file says:
**client sent no required SSL certificate while reading client request headers**
